### PR TITLE
Adding pydeck-earthengine-layers

### PR DESCRIPTION
**Pull request by kylebarron**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by kylebarron** [comment]
I'm struggling to figure out why the CI fails.

**Comment by kylebarron** [comment]
When building locally, I get
```
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform osx-64: {"earthengine-api[version='>=0.1.215']", "pydeck[version='>=0.3.0']"}
```
I'm not sure why `conda build` can't find the noarch versions of these packages.

File: recipes/pydeck-earthengine-layers/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+requirements:
+  host:
+    - earthengine-api
+    - jinja2
+    - pip
+    - pydeck
+    - python
+    - requests
+    - setuptools
```
Comment (scopatz) [inline recipes/pydeck-earthengine-layers/meta.yaml:25]: you don't need pip and setuptools.  Also, do you need all of these packages to build, or just at run time?

Comment (kylebarron) [reply recipes/pydeck-earthengine-layers/meta.yaml:25]: It's a pure-python package, so I'm guessing I don't need these to build

Comment (kylebarron) [reply recipes/pydeck-earthengine-layers/meta.yaml]: @scopatz But as you can see, I was getting failing builds without them

Comment (scopatz) [reply recipes/pydeck-earthengine-layers/meta.yaml]: Weird ok... I guess you do

Comment (kylebarron) [reply recipes/pydeck-earthengine-layers/meta.yaml:25]: I don't know Conda forge packaging very well, so not sure why. But in any case, I reverted the last commit for now.

**Comment by kylebarron** [comment]
Thanks!

**Pull request merged**